testresults/get.go

```diff
@@ -11,26 +11,53 @@ import (
 	"github.com/pkg/errors"
 )
 
+type TestResultsGetOptions struct {
+	CedarOpts timber.GetOptions
+
+	// Request information. See cedar's REST documentation for more
+	// information:
+	// `https://github.com/evergreen-ci/cedar/wiki/Rest-V1-Usage`.
+	TaskID        string
+	DisplayTaskID string
+	TestName      string
+	Execution     int
+}
+
+func (opts *TestResultsGetOptions) validate() error {
+	if err := opts.CedarOpts.Validate(); err != nil {
+		return errors.WithStack(err)
+	}
+
+	if (opts.TaskID == "" && opts.DisplayTaskID == "") || (opts.TaskID != "" && opts.DisplayTaskID != "") {
+		return errors.New("must provide either a task id or a display task id when requesting test results")
```

Comment:
English is a bit vague about "either"; can we phrase this someting like "must provide either ...  but not both"?